money.go

```diff
@@ -198,26 +198,56 @@ func (m *Money) Negative() *Money {
 }
 
 // Add returns new Money struct with value representing sum of Self and Other Money.
-func (m *Money) Add(om *Money) (*Money, error) {
-	if err := m.assertSameCurrency(om); err != nil {
-		return nil, err
+func (m *Money) Add(om ...*Money) (*Money, error) {
+	if len(om) == 0 {
+		panic("At least one Money is required to add")
```

Comment:
I think you can return the original value instead of panic because it is not incorrect mathematically.